pkg/virt-controller/watch/vm_test.go

```diff
@@ -815,20 +819,25 @@ var _ = Describe("VirtualMachine", func() {
 			existingDataVolume.OwnerReferences = nil
 			existingDataVolume.Namespace = "default"
 			existingDataVolume.Status.Phase = cdiv1.Succeeded
+
+			vm, err := virtFakeClient.KubevirtV1().VirtualMachines(vm.Namespace).Create(context.TODO(), vm, metav1.CreateOptions{})
+			Expect(err).To(Succeed())
 			addVirtualMachine(vm)
 			dataVolumeFeeder.Add(existingDataVolume)
 
 			// expect creation called
 			vmiInterface.EXPECT().Create(context.Background(), gomock.Any(), metav1.CreateOptions{}).Do(func(ctx context.Context, arg interface{}, opts metav1.CreateOptions) {
 				Expect(arg.(*v1.VirtualMachineInstance).ObjectMeta.Name).To(Equal("testvmi"))
 			}).Return(vmi, nil)
-			// expect update status is called
-			vmInterface.EXPECT().UpdateStatus(context.Background(), gomock.Any(), metav1.UpdateOptions{}).Do(func(ctx context.Context, arg interface{}, options metav1.UpdateOptions) {
-				Expect(arg.(*v1.VirtualMachine).Status.Created).To(BeFalse())
-				Expect(arg.(*v1.VirtualMachine).Status.Ready).To(BeFalse())
-			}).Return(nil, nil)
+
```

Comment:
nit:empty line

Reply:
Intentional as well to highlight when is the "test" executed